src/components/CompleteApplicationButton/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import Button from "react-bootstrap/Button";
+
+import styles from "./styles.module.css";
+
+export default class CompleteApplicationButton extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      text: props.text,
+      status: props.status,
+    };
+  }
+
+  render() {
+    if (this.state.status === "disabled") {
+      return (
+        <Button className={styles.button + ' ' + styles.disabled} href={"/application"} disabled>
+          {this.state.text}
+        </Button>
+      );
+    } else if (this.state.status === "enabled") {
+      return (
+        <Button className={styles.button} href={"/application"}>
+          {this.state.text}
+        </Button>
+      );
+    } else {
+      return null;
+    }
```

Comment:
Can the status of this ever be anything but "enabled" or "disabled"? If not, I'm wondering if we'd be able to get rid of this case and make the above one an else.